arkouda/pdarrayIO.py

```diff
@@ -222,19 +222,18 @@ def read_all(filenames : Union[str, List[str]],
             d: Dict[str, Union[pdarray, Strings]] = {}
             for item in items:
                 if "seg_string" == item["arkouda_type"]:
-                    d[item["dataset_name"]] = Strings(*item["created"].split("+"))
+                    d[item["dataset_name"]] = Strings.from_parts(*item["created"].split("+"))
```

Comment:
No I don't think there is any reason other than PR drift over time.  This section of code has changed upstream a few times and I'm assuming one of my rebases wrapped the old constructor with the factory method instead of changing to the simpler version of _from_return_msg_